html5lib/html5parser.py

```diff
@@ -78,11 +78,12 @@ def __init__(self, tree=None, tokenizer=tokenizer.HTMLTokenizer,
         self.phases = dict([(name, cls(self, self.tree)) for name, cls in
                             getPhases(debug).items()])
 
-    def _parse(self, stream, innerHTML=False, container="div",
-               encoding=None, parseMeta=True, useChardet=True, **kwargs):
+    def _parse(self, stream, innerHTML=False, container="div", encoding=None,
+               parseMeta=True, useChardet=True, script=True, **kwargs):
 
         self.innerHTMLMode = innerHTML
         self.container = container
+        self.scriptMode = script
```

Comment:
A real nit that doesn't really block anything, but given it really should be seconds to fix it: I'd prefer `scripting` here as well, given the spec calls it the "scripting flag".